client/state/jetpack-connect/reducer/jetpack-auth-attempts.js

```diff
@@ -6,26 +6,29 @@ import { AUTH_ATTEMPS_TTL } from '../constants';
 import { isStale } from '../utils';
 import { JETPACK_CONNECT_COMPLETE_FLOW, JETPACK_CONNECT_RETRY_AUTH } from 'state/action-types';
 import { jetpackAuthAttemptsSchema } from './schema';
+import { keyedReducer } from 'state/utils';
 
-export default function jetpackAuthAttempts( state = {}, action ) {
-	switch ( action.type ) {
+export function authAttempts( state = undefined, { type, attemptNumber } ) {
```

Comment:
I'd assume that if we rename this one from `jetpackAuthAttempts` to `authAttempts`, we'd also rename:

* The current file that this reducer subtree is defined in;
* The name of the state subtree (in the main JPC reducer);
* The name of the corresponding schema import;
* Any selectors that consume this reducer (I think it's only `getAuthAttempts`).

The other thing that we might do here in order to preserve the naming logic would be to name it `jetpackAuthAttempts` instead of `reducer`. This would make it easier to locate if someone is looking for the code that's responsible for this particular reducer subtree.

FWIW, having the above renames in mind, and the purpose of this PR, I'd probably completely refrain from renaming in right now.

Reply:
While I would like to rename and reorganize a large part of the neighboring state tree, nothing is actually renamed here 🙂 

The `default` export continues to be functionally equivalent (although locally it's been named `reducer`). It's at the bottom of the file. This is a _new, named_ export. It's been added, exported, and named so that the functionality of this reducer can be tested, rather than testing the behavior of `keyedReducer`.